docs/2.10.0/docs/concepts/ledger-model/upgrade-model-in-depth.rst

```diff
@@ -258,6 +258,8 @@ Removing a key leads to a validation error.
 
 Changing the type of a key leads to a validation error.
 
+For 2.9, key types may also not include definitions from outside the current package, or ``daml-stdlib``/``daml-prim``
```

Comment:
@dylant-da @remyhaemmerle-da @paulbrauner-da let me know what you think.

Reply:
I find the "or" a bit hard to parse because of the negation. What about the opposite?

"For 2.9, key types can only use definitions from the current package or ``daml-stdlib``/``daml-prim``"

Reply:
Done in upcoming commit.